src/lib.rs

```diff
@@ -282,6 +284,47 @@ pub type DMat3 = DMat3x3;
 /// Matrix of f64s with 4 columns and 4 rows. Alignment 32, size 128.
 pub type DMat4 = DMat4x4;
 
+/// Pads an element to be in an array in a shader.
+///
+/// All elements in arrays need to be aligned to 16s. This automatically aligns your types to 16s.
```

Comment:
```suggestion
/// All elements in arrays need to be aligned to 16 bytes. This automatically aligns your types to 16 bytes.
```

This wording is strange. Should we mention bytes here?